lib/active_merchant/billing/gateways/quickpay/quickpay_v10.rb

```diff
@@ -15,7 +15,7 @@ def initialize(options = {})
       end
 
       def purchase(money, credit_card_or_reference, options = {})
-        MultiResponse.run(true) do |r|
+        process = MultiResponse.run(true) do |r|
```

Comment:
I think it's ok. If the `use_first_response` isn't set to true, I think the default is that the last response becomes the primary response.